### ISSUE TITLE
Test PDBs more

### ISSUE THREAD

**Issue by chrislovecnm** [cockroachdb/cockroach-operator#99]
I need to manually test the PDBs more.  I may have some edge cases.

**Comment by chrislovecnm** [comment]
I need to downsize a CRDB database and see if we remove the PDB as well.

**Comment by chrislovecnm** [comment]
Are we deleting the PDBs as well.

**Comment by chrislovecnm** [comment]
@johnrk this needs to move into the in-progress column, and for some reason, I cannot see it.

**Comment by johnrk** [comment]
Related: https://github.com/cockroachdb/cockroach-operator/issues/135

**Comment by keith-mcclellan** [comment]
@chrislovecnm can we close? The related issue you opened has been complete for a while

**Comment by johnrk** [comment]
Yes, it's been tested. I think PDBs are still being fixed. @chrislovecnm , what's our status on that?

**Comment by keith-mcclellan** [comment]
@chrislovecnm now that GKE testing is working, are we good to move this to acceptance testing?

**Comment by johnrk** [comment]
@chrislovecnm , pod disruption budgets appear to be not working. cc @taroface 

Slack: https://cockroachlabs.slack.com/archives/C0165QKD78F/p1621376687008000
